src/api/__specs__/campaigns-spec.js

```diff
@@ -5,12 +5,21 @@ import * as getJSONP from '../../lib/getJSONP';
 describe('campaigns', () => {
   const callback = sinon.spy();
 
-  beforeEach(() => {
+  before(() => {
     sinon.stub(getJSONP, 'default');
   });
 
-  afterEach(() => {
+  after(() => {
     getJSONP.default.restore();
+  })
+
+  afterEach(() => {
+    try{
+      getJSONP.default.reset();
```

Comment:
Here's the result without using `reset` https://buildkite.com/everyday-hero/react-widgets/builds/1006#6360a0c6-650e-4458-8b53-a5a29520a5d5
The stubs appear to be creating spies (which makes sense when you're asking it spy questions). Not going to look into sinon source tonight :P